jdaviz/utils.py

```diff
@@ -355,6 +355,15 @@ def flux_conversion(spec, values, original_units, target_units):
     return (values * orig_units).to_value(targ_units, equivalencies=eqv)
 
 
+def _convert_surface_brightness_units(data, from_unit, to_unit):
+    try:
+        quantity = data * u.Unit(from_unit)
+        converted_quantity = quantity.to(u.Unit(to_unit))
+        return converted_quantity.value
+    except u.UnitConversionError:
+        pass
```

Comment:
This would cause the function to return `None` but above where this function is called, I do not see any special handling when this happens. What is the purpose of this `pass`?

Reply:
I took the previous feedback into consideration when implementing this. Please refer to the above thread for context. I'd really appreciate hearing your perspective on this. How would you approach this situation, and what do you suggest I do instead?

Reply:
If anything, we should probably just not `try` in the first place for things like moment maps that shouldn't be converted. We can't just let the exception bubble up because that still disables things (moment maps) that should work because they shouldn't be converted)

Reply:
In that case, won't it be cleaner to first check if the data is 2D (what applies to moment map also probably applies to other similar operations that collapses 3D to 2D?) at `if self.current_unit is not None:` and then just not call the function in the first place for those cases? Or is there a reason what isn't possible?